ccm_web/server/src/migrator.ts

```diff
@@ -25,7 +25,7 @@ export const umzug = new Umzug({
     }
   },
   context: sequelize,
-  storage: new SequelizeStorage({ sequelize }),
+  storage: new SequelizeStorage({ sequelize, tableName: 'app_sequelize_meta' }),
```

Comment:
I decided to separate our `migrations` table from the one used by `ltijs`. Let me know what you think.

Reply:
We could rename to whatever, or not make the change at all. Just an idea. I guess I was trying to avoid more table names with camelCasing. And there might be some benefit to keeping `ltijs` even more separate from the rest of what we're doing.

Reply:
https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/127/commits/3f836f18ba5ac8633a33c2d741143c1bf108dc41